libraries/helpers.rb

```diff
@@ -0,0 +1,35 @@
+module ElasticsearchCookbook
+  module Helpers
```

Comment:
Hmm, we could do that, but we're unlikely to call the Helpers from anything but a provider. For example, I wouldn't expect us to call it from a recipe.